Gemfile

```diff
@@ -6,6 +6,7 @@ source 'https://rubygems.org'
 gemspec
 
 group :development, :test do
+  gem 'brakeman', '~> 6.2.1'
```

Comment:
Oh, and when using the pessimistic locking operator I think we shouldn't supply the patch version. 

So `~> 6.2` should be good enough. This keeps the version < 7 but allows minor version updates.